src/main/java/duke/file/Storage.java

```diff
@@ -0,0 +1,142 @@
+package duke.file;
+
+import duke.taskings.Event;
+import duke.taskings.Task;
+import duke.taskings.Todo;
+import duke.taskings.Deadline;
+import duke.messages.Ui;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+
+
+    static final String MISSING_FILE = "MissingFile";
+    static final String FAILED_CREATION = "FailedFileCreation";
+    static final String FILE_INPUT_FAILED = "FailedFileInput";
+    static final String DEADLINE = "D";
+    static final String EVENT = "E";
+    static final String TODO = "T";
+    static final String MARKED = "X";
+
+
+    /**
+     * to check if file exists, then start storing file inputs back into the current tasks array
+     *
+     * @param tasks the array which we would be inserting entries into
+     */
+    public static void initialiseFile(ArrayList<Task> tasks) {
+        try {
+            File file = manageFile();
+            Scanner input = new Scanner(file);
+            while (input.hasNext()) {
+                String[] parsedStoredTask = input.nextLine().split(" \\| ");
+                switch (parsedStoredTask[0]) {
+                case TODO:
+                    if (parsedStoredTask[1].equals(MARKED)) {
+                        tasks.add(new Todo(TODO, parsedStoredTask[2], true));
```

Comment:
It's a bit strange that you have to pass `TODO` as the first argument, and also for the others. Maybe rename the constants to make it more explicit they are related to the storage format.